src/core/icons/fileIcons.ts

```diff
@@ -2489,6 +2489,16 @@ export const fileIcons: FileIcons = {
       ],
     },
     { name: 'roblox', fileExtensions: ['rbxl', 'rbxlx', 'rbxm', 'rbxmx'] },
+    { name: 'luau', fileExtensions: ['luau'] },
+    {
+      name: 'rojo',
+      fileExtensions: ['project.json', 'model.json', 'meta.json'],
```

Comment:
These file-names are too generic to be associated with a specific technology/framework/library.

Many libraries uses `project.json` (e.g nx), `meta.json` (e.g. wordpress) or `model.json`.

Reply:
Created an icon pack for these!